server/server.ts

```diff
@@ -197,6 +193,10 @@ app.listen(PORT, () => {
   }
 });
 
+app.get('/*', (req, res) => {
```

Comment:
why did you change the place of this code block?

Reply:
If on top, will intercept all other commands.